### PR TITLE
Comment out console.log("Not a medium blog :( ");

### PR DESCRIPTION
**Pull request by tog22**
As discussed in our email thread, here's submitting a PR

### CODE REVIEW

File: app/src/js/content.js [hunk 1]
```diff
@@ -23,13 +23,13 @@ window.addEventListener("load", () => {
         mediumScripts.length > 0 ||
         SCRIPT_SRC_REGEX.test(window.location.hostname)
     ) {
-        console.log("Is a medium blog !");
+        console.log("Unlimited Medium extension: medium blog detected");
```
Comment (issammani) [inline app/src/js/content.js]: Let's remove the log all together. This is used mainly for debugging. In the future, I might add a proper logging mechanism and this line would be replaced by a call to `logger.debug` which would print only in dev mode and not in production.

File: app/src/js/content.js [hunk 2]
```diff
@@ -23,13 +23,13 @@ window.addEventListener("load", () => {
...
         browser.runtime.sendMessage({
             type: "medium-blog",
             data: { domain: window.location.host, url: window.location.origin },
         });
     } else {
-        console.log("Not a medium blog :( ");
+        // console.log("Not a medium blog :( "); // disabled to prevent console pollution
```
Comment (issammani) [inline app/src/js/content.js]: Same here. Let's remove it instead of commenting it out.

**Review by issammani — CHANGES_REQUESTED** [review]
Thanks for opening a PR ! I have a few minor comments 😄

**Comment by tog22** [comment]
Sure thing, I just submitted another commit to my repo, which is now showing up in this pull request. Is that the right way to do these things?

**Review by issammani — APPROVED** [review]
Looks good ! Thanks @tog22

**Comment by issammani** [comment]
> Sure thing, I just submitted another commit to my repo, which is now showing up in this pull request. Is that the right way to do these things?

Yes ! That's exactly how you do it 😄 Thanks for the contribution 👍

**Comment by tog22** [comment]
Thanks for the guidance 🙂 This was my first contribution to a project like this.

Thanks also for the extension. Have a great rest of your week.

**Pull request merged**